electron/games.ts

```diff
@@ -58,6 +60,9 @@ class LegendaryGame implements Game {
    * Alias for `Library.listUpdateableGames`
    */
   public static async checkGameUpdates() {
+    if (!User.isLoggedIn()){
```

Comment:
This is no longer a true alias in that case, would recommend modifying the original function.